### ISSUE TITLE
Can't build the next exiftool docker

### ISSUE THREAD

**Issue by xme** [certsocietegenerale/fame_modules#48]
I upgraded my FAME instance and now I can't build my exiftool Docker. Any idea?

```sh /opt/fame/fame/modules/community/processing/exiftool/install.sh: error on 'fame':

Sending build context to Docker daemon   2.56kB

Step 1/7 : FROM alpine:3
 ---> c059bfaa849c
Step 2/7 : ENV EXIFTOOL_VERSION=12.34
 ---> Using cache
 ---> 9c09ceae57ff
Step 3/7 : RUN apk add --no-cache perl perl-utils make
 ---> Using cache
 ---> e6a0fd835c90
Step 4/7 : RUN cpan Archive::Zip
 ---> Running in 153740937ecb
Loading internal logger. Log::Log4perl recommended for better logging

CPAN.pm requires configuration, but most of it can be done automatically.
If you answer 'no' below, you will enter an interactive dialog for each
configuration option instead.

Would you like to configure as much as possible automatically? [yes] yes
Fetching with HTTP::Tiny:
http://www.cpan.org/authors/01mailrc.txt.gz
Reading '/root/.cpan/sources/authors/01mailrc.txt.gz'
............................................................................DONE
Fetching with HTTP::Tiny:
http://www.cpan.org/modules/02packages.details.txt.gz
Reading '/root/.cpan/sources/modules/02packages.details.txt.gz'
  Database was generated on Fri, 21 Jan 2022 16:41:03 GMT
  HTTP::Date not available
.............
  New CPAN.pm version (v2.29) available.
  [Currently running version is v2.28]
  You might want to try
    install CPAN
    reload cpan
  to both upgrade CPAN.pm and run the new version without leaving
  the current session.


...............................................................DONE
Fetching with HTTP::Tiny:
http://www.cpan.org/modules/03modlist.data.gz
Reading '/root/.cpan/sources/modules/03modlist.data.gz'
DONE
Writing /root/.cpan/Metadata
Running install for module 'Archive::Zip'
Fetching with HTTP::Tiny:
http://www.cpan.org/authors/id/P/PH/PHRED/Archive-Zip-1.68.tar.gz
Fetching with HTTP::Tiny:
http://www.cpan.org/authors/id/P/PH/PHRED/CHECKSUMS
Checksum for /root/.cpan/sources/authors/id/P/PH/PHRED/Archive-Zip-1.68.tar.gz ok
'YAML' not installed, will not store persistent state
Configuring P/PH/PHRED/Archive-Zip-1.68.tar.gz with Makefile.PL
Checking if your kit is complete...
Looks good
Generating a Unix-style Makefile
Writing Makefile for Archive::Zip
Writing MYMETA.yml and MYMETA.json
  PHRED/Archive-Zip-1.68.tar.gz
  /usr/bin/perl Makefile.PL -- OK
Running make for P/PH/PHRED/Archive-Zip-1.68.tar.gz
cp lib/Archive/Zip/ZipFileMember.pm blib/lib/Archive/Zip/ZipFileMember.pm
cp lib/Archive/Zip/MemberRead.pm blib/lib/Archive/Zip/MemberRead.pm
cp lib/Archive/Zip/DirectoryMember.pm blib/lib/Archive/Zip/DirectoryMember.pm
cp lib/Archive/Zip/BufferedFileHandle.pm blib/lib/Archive/Zip/BufferedFileHandle.pm
cp lib/Archive/Zip/Archive.pm blib/lib/Archive/Zip/Archive.pm
cp lib/Archive/Zip/Tree.pm blib/lib/Archive/Zip/Tree.pm
cp lib/Archive/Zip/Member.pm blib/lib/Archive/Zip/Member.pm
cp lib/Archive/Zip/FileMember.pm blib/lib/Archive/Zip/FileMember.pm
cp lib/Archive/Zip/FAQ.pod blib/lib/Archive/Zip/FAQ.pod
cp lib/Archive/Zip/StringMember.pm blib/lib/Archive/Zip/StringMember.pm
cp lib/Archive/Zip/NewFileMember.pm blib/lib/Archive/Zip/NewFileMember.pm
cp lib/Archive/Zip/MockFileHandle.pm blib/lib/Archive/Zip/MockFileHandle.pm
cp lib/Archive/Zip.pm blib/lib/Archive/Zip.pm
cp script/crc32 blib/script/crc32
"/usr/bin/perl" -MExtUtils::MY -e 'MY->fixin(shift)' -- blib/script/crc32
Manifying 4 pod documents
  PHRED/Archive-Zip-1.68.tar.gz
  /usr/bin/make -- OK
The current configuration of allow_installing_outdated_dists is 'ask/no', but for this option we would need 'CPAN::DistnameInfo' installed. Please install 'CPAN::DistnameInfo' as soon as possible. As long as we are not equipped with 'CPAN::DistnameInfo' this option does not take effect
Running make test for PHRED/Archive-Zip-1.68.tar.gz
PERL_DL_NONLAZY=1 "/usr/bin/perl" "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef *Test::Harness::Switches; test_harness(0, 'blib/lib', 'blib/arch')" t/*.t
[91m# Calling "unzip" resulted in:
# BusyBox v1.34.1 (2021-11-23 00:57:35 UTC) multi-call binary.
# 
# Usage: unzip [-lnojpq] FILE[.zip] [FILE]... [-x FILE]... [-d DIR]
# 
# Extract FILEs from ZIP archive
# 
# 	-l	List contents (with -q for short form)
# 	-n	Never overwrite files (default: ask)
# 	-o	Overwrite
# 	-j	Do not restore paths
# 	-p	Write to stdout
# 	-t	Test
# 	-q	Quiet
# 	-x FILE	Exclude FILEs
# 	-d DIR	Extract into DIR
# Exit value 256
[0mt/01_init.t ................... ok
[91m
#   Failed test at t/02_main.t line 107.
# Got result:
# unzip: bad archive
# Exit value 256
[0m[91m
#   Failed test at t/02_main.t line 141.
# Got result:
# unzip: bad archive
# Exit value 256
[0m[91m
#   Failed test at t/02_main.t line 162.
# Got result:
# unzip: bad archive
# Exit value 256
[0m[91m
#   Failed test at t/02_main.t line 185.
# Got result:
# unzip: bad archive
# Exit value 256
[0m[91m
#   Failed test at t/02_main.t line 205.
# Got result:
[0m[91m# unzip: bad archive
# Exit value 256
[0m[91m
#   Failed test 'write and test zip file - test write piped'
#   at t/02_main.t line 248.
# Got result:
# unzip: bad archive
# Exit value 256
[0m[91m
#   Failed test 'write and test zip file - test write plain'
#   at t/02_main.t line 248.
# Got result:
# unzip: bad archive
# Exit value 256
[0m[91m
#   Failed test 'write and test zip file - test write piped'
#   at t/02_main.t line 263.
# Got result:
# unzip: bad archive
# Exit value 256
[0m[91m
#   Failed test 'write and test zip file - test write plain'
#   at t/02_main.t line 263.
# Got result:
# unzip: bad archive
# Exit value 256
[0m[91m
#   Failed test 'write and test zip file - test write piped'
#   at t/02_main.t line 280.
# Got result:
# unzip: bad archive
# Exit value 256
[0m[91m
#   Failed test 'write and test zip file - test write plain'
#   at t/02_main.t line 280.
# Got result:
# unzip: bad archive
# Exit value 256
[0m[91m
#   Failed test 'write and test zip file - test write piped'
#   at t/02_main.t line 325.
# Got result:
# unzip: bad archive
# Exit value 256
[0m[91m
#   Failed test 'write and test zip file - test write plain'
#   at t/02_main.t line 325.
# Got result:
# unzip: bad archive
# Exit value 256
[0m[91m
#   Failed test 'write and test zip file - test write piped'
#   at t/02_main.t line 347.
# Got result:
# unzip: bad archive
# Exit value 256
[0m[91m
#   Failed test 'write and test zip file - test write plain'
#   at t/02_main.t line 347.
# Got result:
# unzip: bad archive
# Exit value 256
[0m[91m
#   Failed test at t/02_main.t line 385.
# Got result:
# unzip: bad archive
# Exit value 256
[0m[91m
#   Failed test 'write and test zip file - test write piped'
#   at t/02_main.t line 394.
[0m[91m# Got result:
# unzip: bad archive
# Exit value 256
[0m[91m
#   Failed test 'write and test zip file - test write plain'
#   at t/02_main.t line 394.
# Got result:
# unzip: bad archive
# Exit value 256
[0m[91m
#   Failed test 'write and test zip file - test write piped'
#   at t/02_main.t line 402.
# Got result:
# unzip: bad archive
# Exit value 256
[0m[91m
#   Failed test 'write and test zip file - test write plain'
#   at t/02_main.t line 402.
# Got result:
# unzip: bad archive
# Exit value 256
[0m[91m
#   Failed test 'write and test zip file - test write piped'
#   at t/02_main.t line 411.
# Got result:
# unzip: bad archive
# Exit value 256
[0m[91m
#   Failed test 'write and test zip file - test write plain'
#   at t/02_main.t line 411.
# Got result:
# unzip: bad archive
# Exit value 256
[0m[91m# Looks like you failed 22 tests of 519.
[0mt/02_main.t ................... 
Dubious, test returned 22 (wstat 5632, 0x1600)
Failed 22/519 subtests 
t/03_ex.t ..................... ok
t/04_readmember.t ............. ok
t/05_tree.t ................... ok
t/06_update.t ................. ok
t/07_filenames_of_0.t ......... ok
t/08_readmember_record_sep.t .. ok
t/09_output_record_sep.t ...... ok
t/10_chmod.t .................. ok
t/11_explorer.t ............... ok
t/12_bug_47223.t .............. skipped: Only required on Win32.
t/13_bug_46303.t .............. ok
t/14_leading_separator.t ...... ok
t/15_decrypt.t ................ ok
t/16_decrypt.t ................ ok
t/17_101092.t ................. ok
t/18_bug_92205.t .............. ok
t/19_bug_101240.t ............. ok
t/20_bug_github11.t ........... ok
[91m# Non-zero exit value on reference
# "unzip -t" returned non-zero exit value 256 on file "t/data/zip64.zip"
# (which might be entirely OK on your operating system) and resulted in the
# following output:
# unzip: short read
[0m[91m
#   Failed test 'write and test zip file - test write piped'
#   at t/21_zip64.t line 112.
# Got result:
# Exit value 0
# Expected (more or less) result:
# unzip: short read
# Exit value 256
[0m[91m
#   Failed test 'write and test zip file - test write plain'
#   at t/21_zip64.t line 112.
# Got result:
# Exit value 0
# Expected (more or less) result:
[0m[91m# unzip: short read
# Exit value 256
[0m[91m# Non-zero exit value
# "unzip -t" returned non-zero exit value 256 on file "/tmp/testout-Pybmc.zip"
# (which might be entirely OK on your operating system) and resulted in the
# following output:
# unzip: short read
[0m[91m# Non-zero exit value
# "unzip -t" returned non-zero exit value 256 on file "/tmp/testout-Pybmc.zip"
# (which might be entirely OK on your operating system) and resulted in the
# following output:
# unzip: short read
# Looks like you failed 2 tests of 86.
[0mt/21_zip64.t .................. 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 2/86 subtests 
t/22_deflated_dir.t ........... ok
t/23_closed_handle.t .......... ok
t/24_unicode_win32.t .......... ok
t/25_traversal.t .............. ok
t/26_bzip2.t .................. ok
t/27_symlinks.t ............... ok
t/28_zip64_unsupported.t ...... skipped: Zip64 format is supported.

Test Summary Report
-------------------
t/02_main.t                 (Wstat: 5632 Tests: 519 Failed: 22)
  Failed tests:  347, 358, 368, 378, 386, 403, 405, 413
                415, 424, 426, 438, 440, 448, 450, 468
                474, 476, 482, 484, 490, 492
  Non-zero exit status: 22
t/21_zip64.t                (Wstat: 512 Tests: 86 Failed: 2)
  Failed tests:  65, 67
  Non-zero exit status: 2
Files=28, Tests=911,  4 wallclock secs ( 0.12 usr  0.01 sys +  3.71 cusr  0.92 csys =  4.76 CPU)
Result: FAIL
[91mFailed 2/28 test programs. 24/911 subtests failed.
[0m[91mmake: *** [Makefile:926: test_dynamic] Error 255
[0m  PHRED/Archive-Zip-1.68.tar.gz
  /usr/bin/make test -- NOT OK
//hint// to see the cpan-testers results for installing this module, try:
  reports PHRED/Archive-Zip-1.68.tar.gz
The command '/bin/sh -c cpan Archive::Zip' returned a non-zero code: 1```

**Comment by gaelmuller** [comment]
I was not able to reproduce. Are you still having this issue ?

**Comment by Heat-Miser** [comment]
Hello

same problem here, it seems that adding "-f" after cpan to the "RUN cpan Archive::Zip" solves the problem.

The tests fail but the modules is working. No idea why.

**Comment by gaelmuller** [comment]
Fixed in #49
